Apps/W1/HybridGP/app/src/Migration/Vendors/GPVendorMigrator.codeunit.al

```diff
@@ -3,13 +3,12 @@ codeunit 4022 "GP Vendor Migrator"
     TableNo = "GP Vendor";
 
     var
+        HelperFunctions: Codeunit "Helper Functions";
```

Comment:
Lets not introduce global variables. It is better to keep them local, it is easier to read and we are not introducing a global state.